core/tests/protractor-browserstack.conf.js

```diff
@@ -0,0 +1,219 @@
+  //
+  // There are three ways to specify how to use Selenium. Specify one of the
+  // following:
+  //
+  // 1. seleniumServerJar - to start Selenium Standalone locally.
+  // 2. seleniumAddress - to connect to a Selenium server which is already
+  //    running.
+  // 3. sauceUser/sauceKey - to use remote Selenium servers via SauceLabs.
+  //
+  // If the chromeOnly option is specified, no Selenium server will be started,
+  // and chromeDriver will be used directly (from the location specified in
+  // chromeDriver)
+
+  // The location of the selenium standalone server .jar file, relative
+  // to the location of this config. If no other method of starting selenium
+  // is found, this will default to
+  // node_modules/protractor/selenium/selenium-server...
+  seleniumServerJar: null,
+  // The port to start the selenium server on, or null if the server should
+  // find its own unused port.
+  seleniumPort: null,
+  // Chromedriver location is used to help the selenium standalone server
+  // find chromedriver. This will be passed to the selenium jar as
+  // the system property webdriver.chrome.driver. If null, selenium will
+  // attempt to find chromedriver using PATH.
+  chromeDriver: './selenium/chromedriver',
+  // If true, only chromedriver will be started, not a standalone selenium.
+  // Tests for browsers other than chrome will not run.
+  chromeOnly: false,
+  // Additional command line options to pass to selenium. For example,
+  // if you need to change the browser timeout, use
+  // seleniumArgs: ['-browserTimeout=60'],
+  seleniumArgs: [],
+
+  // If sauceUser and sauceKey are specified, seleniumServerJar will be ignored.
+  // The tests will be run remotely using SauceLabs.
+  sauceUser: null,
+  sauceKey: null,
+
+  // The address of a running selenium server. If specified, Protractor will
+  // connect to an already running instance of selenium. This usually looks like
+  // seleniumAddress: 'http://localhost:4444/wd/hub'
+  seleniumAddress: 'http://hub-cloud.browserstack.com/wd/hub',
+
+  // The timeout for each script run on the browser. This should be longer
+  // than the maximum time your application needs to stabilize between tasks.
+  // (Note that the hint tooltip has a 60-second timeout.)
+  allScriptsTimeout: 180000,
+
+  // ----- What tests to run -----
+  //
+  // When run without a command line parameter, all suites will run. If run
+  // with --suite=smoke, only the patterns matched by that suite will run.
+  suites: {
+    full: [
+      'protractor/accessibility.js',
+      'protractor/learnerFlow.js',
+      'protractor/loginFlow.js',
+      'protractor_mobile/navigation.js',
+      'protractor_mobile/playerFlow.js',
+      'protractor/ratings.js',
+      'protractor/subscriptionsFlow.js'
+    ],
+
+    accessibility: [
+      'protractor/accessibility.js'
+    ],
+
+    learner: [
+      'protractor/learnerFlow.js'
+    ],
+
+    library: [
+      'protractor/libraryFlow.js'
```

Comment:
Why is this here but not in "full" above?

Actually probably better if "full" was `protractor/*.js` and `protractor_mobile/*.js`? Otherwise we'll miss stuff by mistake (like we did for the learner dashboard tests some time back).

Reply:
Oh, that's interesting. But it is vulnerable to accidentally leaving out things, e.g. libraryFlow above.

Still, agreed with wanting to make the tests run in parallel if possible. Maybe add a comment here to explain why we specify each thing individually and a note to developers to make sure that there's parity between the two lists. (Or a simple frontende/backend test that loads the config file somehow and verifies that this condition holds.)

Should we be doing something similar in our existing protractor.conf.js file, to?

Reply:
I will add a comment over here. Also, in case of our tests, we follow a different approach on Travis -- we call in each suite individually:
`bash scripts/run_e2e_tests.sh --suite=<testSuite> --prod_env`
to make it run parallel.
Initially, I gave this approach a try for browserstack as well but the tests flaked out since we were parallelizing them on the Travis end and not exactly on the browserstack end.

Reply:
Ah OK. Mention that in the comment as well, or maybe in a similar comment in protractor.conf.js. That way developers will be able to understand why the two approaches are different. (It might also be nice to make them the same and perhaps modify the e2e script as needed, but that's your call.)

Reply:
Mentioned this as a comment, done. Thanks!